tests/DistortionTests.cpp

```diff
@@ -124,17 +124,26 @@ TEST(transverse, removeDistortion_AlternatingOnes) {
   EXPECT_NEAR(uy, 7.5, 1e-8);
 }
 
-TEST(Radial, testRemoveDistortion) {
-  csm::ImageCoord imagePt(0.0, 4.0);
+TEST(Radial, testUndistortDistort) {
```

Comment:
I'm a little concerned about this test change but it's likely fine(?)

Reply:
I will tab those in a bit. I left the text with original indentation to make it easier to see the diff. (Sloppy, yes.) 

Why is the distortion test a bit concerning? I on purpose chose a value to make the test fail before, because it was illustrating an inconsistency. I can easily go back to the previous pixel value, though likely any value will work as good as any other value, I am guessing.

I can also push down the tolerance value there if needed.

Reply:
The code was not doing either distortion or understortion if the norm of the pixel was less than tolerance.

If the pixel is a little above tolerance, then distortion gets applied. The produced output pixel ends up a little below tolerance. Then undistortion is not applied.

This is a little silly, but a concrete example will help. Say x = 0.012, and tol = 0.01, and according to the formula distort(x) = 0.009. Then, normally, undistort(0.009) has to give back 0.012, but since 0.009 < tol, one gets undistort(0.009) = 0.009, which is the wrong answer. Granted, normally tol is small, so this does not matter much.

Reply:
I fixed the indentation.